### PR TITLE
fix #192, order admin fields, fix help_text

### LINKED ISSUE #192: Filter localities in admin panel
Right now, all localities are loaded in all views. This is SLOW; there are tons, and we do a reverse string lookup to make them render meaningfully.

Better is to filter OUT zip-codes when displaying in the admin; they are never appropriate for our various locality filters (ballots, committees, etc).

### PR DESCRIPTION
**Pull request by mikeubell**
I decided to put the fields in the Admin interface in a more reasonable order, e.g. Name first.
I dropped the ballot_type and photo_url from the referendum page as the latter is set by the initializer and the latter does not make much sense.  Let me know if I missed any significant fields from this or other pages.

### CODE REVIEW

**Comment by bcipolli** [comment]
@mikeubell Sounds great! Could you add screen shots? It would make it easier to see what you've got, rather than pulling and running the code myself.

**Comment by mikeubell** [comment]
Forgot to run make test.... new push coming.

**Comment by mikeubell** [comment]
You want all the admin pages?

**Comment by bcipolli** [comment]
just any key changes that you wanna show off :)

**Comment by mikeubell** [comment]
untitled 6
untitled 5
untitled 4
untitled 3
untitled 2
untitled

File: ballot/admin/ballot.py [hunk 1]
```diff
@@ -2,4 +2,9 @@
 
 from ..models import ballot as models
 
-admin.site.register(models.Ballot)
+
+class BallotAdmin(admin.ModelAdmin):
+    exclude = ['locality']
```
Comment (bcipolli) [inline ballot/admin/ballot.py]: Why exclude `locality` from Ballot? I believe this is one of the key fields...

File: ballot/admin/referendum.py [hunk 2]
```diff
@@ -2,4 +2,10 @@
 
 from ..models import referendum as models
 
-admin.site.register(models.Referendum)
+
+class ReferendumAdmin(admin.ModelAdmin):
+    fields = ('name', 'number', 'ballot',
+              'website_url', 'facebook_url', 'twitter_url')
```
Comment (bcipolli) [inline ballot/admin/referendum.py:10]: Can you add comments to each of these, as to why you're doing this? 

A big major downside of listing fields explicitly like like this is, if new fields are added to the model, then they must also be added here (which isn't true for most models).

Here and elsewhere, let's add an `assert` (or a test) that will notify us if things change.
`assert len(fields) - len(Referendum._meta.get_fields()) == some_num, "Make sure there are no new fields."`

File: ballot/models/referendum.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ class Referendum(BallotItem, SocialMediaMixin):
     # TODO: Set up a save() event, or overload save(), to auto-populate
     #   BallotItemSelections (YES/NO)
     name = models.CharField(
-        max_length=255, help_text='The referendum number or the name '
+        max_length=255, help_text='The descripteve referendum name or the name '
```
Comment (bcipolli) [inline ballot/models/referendum.py]: I find this to also be confusing (and misspelled). How about `A short referendum title or description`? Should we change this to `title`?

**Comment by bcipolli** [comment]
@mikeubell This is very nice! I've requested a few code changes, and you'll need to rebase on top of the current master and resolve conflicts.

If you need any help with that, please add me as a collaborator on your repo and I can take care of it!

File: ballot/admin/referendum.py (hunk 2 above)
Comment (mikeubell) [inline ballot/admin/referendum.py:10]: I looked for some way around this.  The ordering in the model seems to have no effect on the ordering on the admin page.  Is there some sort of “start_with” directive for admin?  Or maybe an ordering directive?

Comment (bcipolli) [inline ballot/admin/referendum.py:10]: I didn't see any other way as well. I think as long as we add the `assert` I wrote to each admin where we do this reordering, or add a test for each admin where we do this reordering, it's fine--we'll at least notice if a field is added/removed and the admin not updated.

**Comment by bcipolli** [comment]
It would be great to have these asserts/tests done before merging the branch adding the `alias` field, as that will provide a nice test case for this.

I will work on this now.

**Comment by mikeubell** [comment]
I have that mostly done. I should be able to finish tomorrow.  Would be
best if you worked on something else.

**Comment by bcipolli** [comment]
Ah, that's great--glad to work on other things. Thanks for the note!

File: ballot/admin/ballot.py (hunk 1 above)
Comment (mikeubell) [inline ballot/admin/ballot.py]: I guess it was another zipcode list that took a long time.  I will take it out.  Why is a ballot locality a zipcod anyway?  Shouldn't it be a city or some other jurisdiction?

Comment (bcipolli) [inline ballot/admin/ballot.py]: It's not; it's a generic `Locality`, which can refer to any of those.

We need to filter the `Locality` values that are shown in the dropdown, not to eliminate the dropdown. I would suggest filtering out ZipCode instances.

Perhaps something like this:
https://stackoverflow.com/questions/6581520/filtering-dropdown-values-in-django-admin#answer-6582108

Comment (mikeubell) [inline ballot/admin/ballot.py]: So every zipcode has its on ballot???  I thought a ballot was attached to a jurisdiction, e.g. Oakland.  What values do you want to filter out of the zip code drop down?  It should have cities and counties in it I think.

**Comment by mikeubell** [comment]
Ben: This seems to have gone quiet?  If I don't hear objections I will rebase and merge it in the next day.

**Comment by bcipolli** [comment]
This looks good to me. After seeing the asserts, they probably fit better as tests.. but we can do that anytime. Better to merge and move forward!

**Pull request merged**